index.js

```diff
@@ -129,7 +180,15 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
+function getAllMoviesReleasedAtOrBeforeYear(movies, year) {
```

Comment:
Here I think we should have considered `movies[i].released` might have more data than just a year, which also might possibly not be a `Number`.